cmd/main.go

```diff
@@ -0,0 +1,172 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+
+	cobbler "github.com/ContainerSolutions/cobblerclient"
+)
+
+var config = cobbler.ClientConfig{
+	Url:      "http://localhost:25151",
+	Username: "cobbler",
+	Password: "password",
+}
+
+func main() {
+	c := cobbler.NewClient(http.DefaultClient, config)
+	_, err := c.Login()
+	if err != nil {
+		fmt.Printf("Error logging in: %s\n", err)
+	}
+
+	fmt.Printf("Token: %s\n", c.Token)
+
+	d := cobbler.Distro{
+		Name:      "Test",
+		Breed:     "Ubuntu",
+		OSVersion: "trusty",
+		Arch:      "x86_64",
+		Kernel:    "/var/www/cobbler/ks_mirror/Ubuntu-14.04/install/netboot/ubuntu-installer/amd64/linux",
+		Initrd:    "/var/www/cobbler/ks_mirror/Ubuntu-14.04/install/netboot/ubuntu-installer/amd64/initrd.gz",
+	}
+
+	fmt.Println("Creating a Distro")
+	newDistro, err := c.CreateDistro(d)
+	if err != nil {
+		fmt.Printf("Error creating distro: %s\n", err)
+	}
+
+	fmt.Printf("New Distro: %+v\n", newDistro)
```

Comment:
This line and the rest below it that reference to `newDistro` will fail if the distro is not created.